### ISSUE TITLE
Dictybase xrefs in the UniProt download

### ISSUE THREAD

**Issue by katewarner** [glygener/glygen-issues#1583]
It looks like there's a large decrease in the number of dictybase xrefs in the 2024_03 download. Is this a problem with the download or has UniProt removed a large number of dictybase xrefs? 

FYI @rykahsay 

```
[k.warner1@glygen-vm-dev ~]$ grep -c 'dictybase' /data/projects/glygen/downloads/ebi/current/uniprot-proteome-dictyostelium-discoideum.nt
32610
[k.warner1@glygen-vm-dev ~]$ grep -c 'dictybase' /data/projects/glygen/downloads/ebi/2024_04_08/uniprot-proteome-dictyosteliu
m-discoideum.nt
40713
```

**Comment by pv47** [comment]
@katewarner think there was a problem only in 2024_03, fixed in 204_04

> zgrep -c 'dictybase' previous_releases/2024_0*/uniprot-proteome-dictyostelium-discoideum.nt.gz
previous_releases/2024_01/uniprot-proteome-dictyostelium-discoideum.nt.gz:40713
previous_releases/2024_02/uniprot-proteome-dictyostelium-discoideum.nt.gz:40713
previous_releases/2024_03/uniprot-proteome-dictyostelium-discoideum.nt.gz:32610
> zgrep -c 'dictybase' 2024_04/uniprot-proteome-dictyostelium-discoideum.nt.gz: 40716

**Comment by katewarner** [comment]
@rykahsay it looks like there was a problem with the Dictybase xrefs in the last download for the 2.6 release which has been fixed in the latest UniProt download (see comments above). This only relates to dictyostelium-discoideum. I'm not sure how you want to proceed? Do you want me to download this data from the again later release?

**Comment by rykahsay** [comment]
It is too late to change EBI version for 2.6, at this point ---> this will get fixed in 2.7, and we can tolerate it for now

**Comment by katewarner** [comment]
Okay thanks. I'll close for now and double check the xrefs are correct for the next release
